src/client/tablet_client.cc

```diff
@@ -543,11 +542,13 @@ bool TabletClient::GetTermPair(uint32_t tid, uint32_t pid, uint64_t& term, uint6
     return true;
 }
 
-bool TabletClient::GetManifest(uint32_t tid, uint32_t pid, ::openmldb::api::Manifest& manifest) {
+bool TabletClient::GetManifest(uint32_t tid, uint32_t pid, ::openmldb::common::StorageMode storage_mode,
```

Comment:
same here. whenever we have `tid` and `pid`, we definitely know the `storage_mode` by looking up the tables_  in tablet.